subsys/bluetooth/controller/ll_sw/ull_peripheral.c

```diff
@@ -418,6 +418,7 @@ void ull_periph_setup(struct node_rx_pdu *rx, struct node_rx_ftr *ftr,
 	conn_offset_us -= EVENT_TICKER_RES_MARGIN_US;
 	conn_offset_us -= EVENT_JITTER_US;
 	conn_offset_us -= ready_delay_us;
+	conn_offset_us -= lll->periph.window_widening_periodic_us;
```

Comment:
Should it be something like below?

```suggestion
	conn_offset_us -= 
	DIV_ROUND_UP(((lll_clock_ppm_local_get() +
				   lll_clock_ppm_get(conn->periph.sca)) *
(win_offset * CONN_INT_UNIT_US + win_delay_us)), USEC_PER_SEC);
```

Reply:
As a simplification, yes, agreed. Please add an inline comment explain the same.

```
/*
NOTE: Correct window widening for the first connection will be:

conn_offset_us -= 
	DIV_ROUND_UP(((lll_clock_ppm_local_get() +
				   lll_clock_ppm_get(conn->periph.sca)) *
(win_offset * CONN_INT_UNIT_US + win_delay_us)), USEC_PER_SEC);

But, as currently in the implementation the drift compensation uses the `lll->periph.window_widening_periodic_us` value.
We may as well use that value here as well. Adding another value for LLL to use seems overkill for this one case.

*/
```

Reply:
Comment added